Apps/Admin/Server/Services/SupportService.cs

```diff
@@ -211,6 +207,52 @@ public async Task BlockAccessAsync(string hdid, IEnumerable<DataSource> dataSour
             await this.patientRepository.BlockAccess(blockAccessCommand, ct).ConfigureAwait(true);
         }
 
+        private async Task<IList<PatientSupportDependentInfo>> GetDependentsAsync(string delegateHdid, CancellationToken ct)
+        {
+            List<PatientSupportDependentInfo> dependents = new();
+
+            IList<ResourceDelegate> resourceDelegates = await this.SearchDependents(delegateHdid);
+            foreach (ResourceDelegate resourceDelegate in resourceDelegates)
+            {
+                PatientModel? dependentPatient = await this.GetPatientAsync(PatientIdentifierType.Hdid, resourceDelegate.ResourceOwnerHdid, ct);
+                if (dependentPatient != null)
+                {
+                    PatientSupportDependentInfo dependentInfo = this.autoMapper.Map<PatientModel, PatientSupportDependentInfo>(dependentPatient);
+                    dependentInfo.ExpiryDate = resourceDelegate.ExpiryDate;
```

Comment:
Would this be not cleaner by creating a mapper util and pass in ExpiryDate like it is done with email via UserFeedbackMapUtils.cs?

Reply:
If we were performing this conversion in multiple places, I would agree, but as this is the only place, I don't think the extra overhead is worth it.